src/applications/simple-forms/21-4138/tests/e2e/fixtures/mocks/featureToggles.json

```diff
@@ -2,14 +2,6 @@
   "data": {
     "type": "feature_toggles",
     "features": [
-      {
-        "name": "profile_show_profile_2.0",
-        "value": false
-      },
-      {
-        "name": "form214138",
-        "value": true
-      },
```

Comment:
Why are we removing these? Looks like `"form214138"` is a duplicate but what about `"profile_show_profile_2.0"`?

Reply:
That was actually something that should have been removed when I first generated the app skeleton. Not sure why, that's what the guide says to do. My assumption is that it's outdated.